artist_query/lib/artist_query.ex

```diff
@@ -0,0 +1,51 @@
+defmodule ArtistQuery do
+  def main(args) do
+    start_query(args)
+  end
+
+  defp start_query(music_file) do
+    artist = String.upcase(prompt)
+    load_songs(music_file)
```

Comment:
I'd probably add a newline above here to visually distinguish this main block from the assignment above.